### PR TITLE
EI-603 Add some exports to listAllTwins

### PR DESCRIPTION
**Pull request by zoltaniotic**

### CODE REVIEW

File: src/listAllTwins.ts [hunk 1]
```diff
@@ -8,8 +8,9 @@ import {
 import { Headers, Range, Limit, Offset } from "./client/iotics/api/common_pb";
 import { getShortUUID } from "./helpers";
 
-const PAGE_LENGTH = 100;
+export const LIST_ALL_TWINS_PAGE_LENGTH = 100;
 const STATUS_OK = "OK";
+export const LIST_ALL_TWINS_STATUS_END = "END";
```
Comment (Evolter) [inline src/listAllTwins.ts:13]: You may want to add a line to [change log](https://github.com/Iotic-Labs/iotics-grpc-client-ts/blob/main/CHANGELOG.md), something like:
```markdown
## [Unreleased]
### Changed 
- Constants in `listAllTwins` for better pagination handlings. 
```

File: src/listAllTwins.ts [hunk 2]
```diff
@@ -8,8 +8,9 @@ import {
 import { Headers, Range, Limit, Offset } from "./client/iotics/api/common_pb";
 import { getShortUUID } from "./helpers";
 
-const PAGE_LENGTH = 100;
+export const LIST_ALL_TWINS_PAGE_LENGTH = 100;
 const STATUS_OK = "OK";
```
Comment (Evolter) [inline src/listAllTwins.ts:12]: Maybe change to `LIST_ALL_TWINS_STATUS_OK` for consistency?

File: src/listAllTwins.ts [hunk 3]
```diff
@@ -96,14 +97,18 @@ const callBackListAllTwins = (
...
                 client,
                 clientAppId,
                 metadata,
                 page + 1,
                 listCallback
             );
+        } else {
+            listResult.status = LIST_ALL_TWINS_STATUS_END;
+            listResult.results = undefined;
+            emit(listResult);
```
Comment (Evolter) [inline src/listAllTwins.ts:111]: Not entirely sure, but shouldn't there be `emit(END);` at the end as well?

Comment (zoltaniotic) [reply src/listAllTwins.ts:111]: Changed

File: src/listAllTwins.ts (hunk 2 above)
Comment (zoltaniotic) [reply src/listAllTwins.ts]: Changed

File: src/listAllTwins.ts (hunk 1 above)
Comment (zoltaniotic) [reply src/listAllTwins.ts]: Added

**Review by Evolter — APPROVED** [review]

**Pull request merged**